actions/rm-runner-ip/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -eu
+
+# remove the runner IP from the list of authorized IPs
+CUR_IPS=$(gcloud container clusters describe "${K8_CLUSTER}" --format json | \
+    jq '[ .masterAuthorizedNetworksConfig.cidrBlocks[] | .cidrBlock ]')
+RUNNER_IP=$(curl 'https://api.ipify.org/?format=text')
+RESTORE_IPS=$(echo "${CUR_IPS}" | \
+    jq -r --arg ip "${RUNNER_IP}" '[ .[] | select(startswith($ip) | not) ] | unique | join(",")')
+
+# restore the original list of authorized IPs if they exist
+gcloud container clusters update "${K8_CLUSTER}" \
```

Comment:
Adding the IP was in a retry loop, but removing wasn't. If this fails, is it a security risk?

Reply:
I don't think it's a large security risk, since you still need to be authenticated with the cluster to modify any cluster configuration, but it's probably worth adding it in a loop at minimum.